### PR TITLE
Add required timeout import

### PR DESCRIPTION
**Pull request by StrangeQuark87**
Add missing import causing errors with Windows DNS

### CODE REVIEW

**Comment by ezr-ondrej** [comment]
ok to test

**Comment by ekohl** [comment]
Looks like this was introduced in 168ee4e64718eed50e3e140d0fd2d54db0999b56. Even Ruby 2.0.0 documentation states:
> Previous versions didn't use a module for namespacing, however timeout is provided for backwards compatibility. You should prefer Timeout#timeout instead.

So this looks correct. We do require an issue to be created. Could you [create one](https://projects.theforeman.org/projects/smart-proxy/issues/new)?

File: modules/dns_dnscmd/dns_dnscmd_main.rb [hunk 1]
```diff
@@ -37,7 +38,7 @@ def execute(*args, **keyword_args)
       std_in = std_out = std_err = nil
       real_cmd = ['c:\Windows\System32\dnscmd.exe', @server] + args
       begin
-        timeout(tsecs) do
+        Timeout.timeout(tsecs) do
```
Comment (ekohl) [inline modules/dns_dnscmd/dns_dnscmd_main.rb:41]: This was addressed in aa25357d4404908d5e5707e767ecd798b754d7d1.

File: modules/dns_dnscmd/dns_dnscmd_main.rb [hunk 2]
```diff
@@ -1,4 +1,5 @@
 require 'open3'
+require 'timeout'
```
Comment (ekohl) [inline modules/dns_dnscmd/dns_dnscmd_main.rb:2]: Done in https://github.com/theforeman/smart-proxy/pull/842

**Pull request closed without merging**